resources/validations/docs/adr/0007-web-frontend.md

```diff
@@ -0,0 +1,30 @@
+# 7. Validations User Interface
```

Comment:
NB: Incredibly well-written and nice work, just wanted to give credit where it is due.